### PR TITLE
Add USD value in Transfer Modal

### LINKED ISSUE #1452: Transfer modal: show the equivalent amount in $ when doing a transfer
When i do a transfer i want to see how much the SBD or Steem that i want to send is worth in $

@pareshUD do you have a design for this?

### PR DESCRIPTION
**Pull request by jm90m**
Fixes #1452 

Changes:
- Display current USD amount of transfer

screen shot 2018-02-01 at 9 39 44 pm

### CODE REVIEW

**Comment by bonustrack** [comment]
It would be great to show the price like this: 
image
With text in grey color. If you cant do this, we need @pareshUD ideas

**Comment by jm90m** [comment]
yeah thats fine I'll see what i can do

**Comment by jm90m** [comment]
Should we also display the 'Current USD value of this transfer' text

**Comment by jm90m** [comment]
@pareshUD would be good to have some ideas also

**Comment by bonustrack** [comment]
We dont need to show `Current USD value of this transfer` message. Just showing the estimation is self explanatory

**Comment by pareshUD** [comment]
transfer
transfer2

@jm90m @bonustrack Please try this.

**Comment by Sekhmet** [comment]
@pareshUD Fabien suggested displaying value inside of input:
https://github.com/busyorg/busy/pull/1455#issuecomment-362841084

**Comment by pareshUD** [comment]
yes @Sekhmet But I tried to display it in a better way.. Like having them on the far right.. But then I am finding the issue with mobile version when the screen resolution is smaller.. So I might reworked on this

**Comment by bonustrack** [comment]
We can use the same position as @pareshUD suggested, should be easier to implement anyway.

**Comment by jm90m** [comment]
yea this is way easier to implement

**Comment by jm90m** [comment]
Updated with @pareshUD & @bonustrack  style

File: src/client/locales/default.json [hunk 1]
```diff
@@ -313,5 +313,6 @@
   "login_to_busy": "Login to Busy",
   "login_modal_description": "Login with your Steem account using SteemConnect to enjoy Busy at 100%",
   "login_with_steemconnect": "Login with SteemConnect",
-  "follows_you": "Follows you"
+  "follows_you": "Follows you",
+  "transfer_current_usd_amount": "Current USD value of this transfer: ${amount}"
```
Comment (Sekhmet) [inline src/client/locales/default.json]: This isn't used anymore so I think we don't have to touch translations.

Comment (jm90m) [reply src/client/locales/default.json]: yeah ill remove it

File: src/client/locales/default.json [hunk 2]
```diff
@@ -313,5 +313,5 @@
   "login_to_busy": "Login to Busy",
   "login_modal_description": "Login with your Steem account using SteemConnect to enjoy Busy at 100%",
   "login_with_steemconnect": "Login with SteemConnect",
-  "follows_you": "Follows you"
+  "follows_you": "Follows you",
```
Comment (Sekhmet) [inline src/client/locales/default.json]: Technically it's not correct JSON with trailing comma, but I think Crowdin will work with this anyway.

Comment (jm90m) [reply src/client/locales/default.json]: no you're right, I'll remove it

Comment (jm90m) [reply src/client/locales/default.json]: let me remove it

**Comment by pareshUD** [comment]
Can we remove the - from $.. It looks like the balance is in negative at the moment. @jm90m @bonustrack 

screen shot 2018-02-08 at 1 52 35 am

**Comment by bonustrack** [comment]
I think it's cool with the `~` it's not a `-`, it's an estimation, it make sense to have it, we have it also in vote value.

**Comment by jm90m** [comment]
yeah I agree with @bonustrack I added the `~` to signify that its an estimation

**Comment by pareshUD** [comment]
How about we use "=" instead?

**Comment by jm90m** [comment]
you can't use `=` that would signify that it is that exact value, this is an estimation

**Pull request merged**